### ISSUE TITLE
Need to set fixed height, otherwise the content is not visible (only the header) (Android)

### ISSUE THREAD

**Issue by goinnovative** [react-native-community/react-native-tab-view#386]
### Expected behaviour
See the content of the tabs.

### Actual behaviour
Only see the header.

### Code sample, screenshots
```
import React from 'react';
import { View, Text, Dimensions } from 'react-native';
import { TabViewAnimated, TabBar } from 'react-native-tab-view';
import Contact from './Contact'
import Location from './Location'
import Coupon from './Coupon'
import Map from './Map'
import Settings from '../../../lib/settings'

import styles from './styles'

const window = Dimensions.get('window');

class Tabs extends React.PureComponent {
  state = {
    index: 0,
    routes: [
      { key: 'location', title: 'Location' },
      { key: 'map', title: 'Karte' },
      { key: 'contact', title: 'Kontakt' },
    ],
  };

  componentDidMount() {
    if(this.props.dish.coupon) {
      this.setState({
        routes: [
          { key: 'location', title: 'Location' },
          { key: 'coupon', title: 'Gutschein' },
          { key: 'map', title: 'Karte' },
          { key: 'contact', title: 'Kontakt' },
        ]
      });
    }
  }

  handleIndexChange = index => {
    this.setState({ index });
    this.props.onShowTab(this.state.routes[index].key);
  };

  renderHeader = props => <TabBar
    getLabelText={({ route }) => route.title}
    style={{backgroundColor: '#fff' }}
    indicatorStyle={{backgroundColor: Settings.primaryColor }}
    labelStyle={{ color: Settings.secondaryColor, fontSize: 12 }}
    {...props}
  />;

  renderScene = ({route}) => {
    return <View style={{height: 100, width: 100, backgroundColor: 'red'}} />;
    switch(route.key) {
      case 'location': return <Location {...this.props} />;
      case 'contact': return <Contact {...this.props} />;
      case 'coupon': return <Coupon {...this.props} />;
      // case 'map': return <Map {...this.props} />;
      case 'map': return null;
    }
  };

  render() {
    return (
      <TabViewAnimated
        navigationState={this.state}
        renderScene={this.renderScene}
        renderHeader={this.renderHeader}
        onIndexChange={this.handleIndexChange}
      />
    );
  }
}

export default Tabs;

```

### What have you tried
If I set the height of the TabViewAnimated manually I see the content. Otherwise under the header is nothing.

**Comment by huangciyin** [comment]
have the same problem too.

**Comment by fullhdpixel** [comment]
Does anyone have a solution for this in mind?

**Comment by satya164** [comment]
It's impossible to repro with the given code. Please create an example on https://snack.expo.io so I can actually repro this. I have never encountered this and it's most likely something with your own components rather than the library.

**Comment by YogiGaxon** [comment]
I am facing almost same issue of tab content height. When i user TabView in ScrollView view tab bar is showing but tab content is not showing. Can anybody help me to resolve this issue.
Below is my code when i am facing issue.

<ScrollView>
     <View style={{flex: 1}}>
        <TabView
        navigationState={this.state}
        renderScene={SceneMap({
          first: FirstRoute,
          second: SecondRoute,
        })}
        onIndexChange={index => this.setState({ index })}
        initialLayout={{ width: Dimensions.get('window').width }}
      />
      </View>
</ScrollView>

Please help me to resolve this issue.

**Comment by satya164** [comment]
Hey, I just released a new alpha `2.0.0-alpha.0` of the library. It's rewritten using [react-native-gesture-handler](https://github.com/kmagiera/react-native-gesture-handler) and [react-native-reanimated](https://github.com/kmagiera/react-native-reanimated) addresses a many platform specific bugs and performance problems. The documentation is updated as well.

Please try the new version and see if it addresses your issue. If not, please open a new issue following the issue template.
